src/huggingface_hub/inference/_client.py

```diff
@@ -146,6 +146,9 @@ class InferenceClient:
         headers (`Dict[str, str]`, `optional`):
             Additional headers to send to the server. By default only the authorization and user-agent headers are sent.
             Values in this dictionary will override the default values.
+        bill_to (`str`, `optional`):
+            The billing account to use for the requests. By default the requests are billed on the user's account.
+            Requests can be billed to any organization the user is a member of as long as billing is enabled for that organization.
```

Comment:
It might be obvious but let's mention that we cannot bill to another user account 😄 or we can change the wording with something like "Request can **only** be billed to ..."
```suggestion
            The billing account to use for the requests. By default the requests are billed on the user's account.
            Requests can only be billed to an organization the user is a member of, and the billing should be enabled for that organization.
```